src/js/controller/controller.js

```diff
@@ -1025,15 +1040,29 @@ Object.assign(Controller.prototype, {
         };
 
         // View passthroughs
-        this.resize = _view.resize;
-        this.getSafeRegion = _view.getSafeRegion;
-        this.setCaptions = _view.setCaptions;
+        if (__HEADLESS__) {
+            // These should be overridden by the app using the headless player
+            this.resize = this.setCaptions = function() {};
```

Comment:
is there a mechanism in place for these overrides or is this something for the future?

Reply:
The features these methods interact with are not supported in the headless player.

Reply:
Makes sense